frontend/ui/widget/menu.lua

```diff
@@ -765,7 +765,9 @@ override this function to process the item selected in a different manner
 ]]--
 function Menu:onMenuSelect(item)
     if item.sub_item_table == nil then
-        self.close_callback()
+        if self.close_callback then
```

Comment:
Just a crash when i tried first to do without a `self.hist_menu.close_callback`. There's one other such test in menu.lua, so this one here must have been forgotten.